pkg/ddevapp/ddevapp.go

```diff
@@ -1958,6 +1958,14 @@ func (app *DdevApp) DockerEnv() {
 		dbPortStr = app.HostDBPort
 	}
 
+	// DDEV_DATABASE_PROTOCOL can be use for connection URLs in the format of protocol//[hosts][/database][?properties]
```

Comment:
```suggestion
	// DDEV_DATABASE_PROTOCOL can be use for connection URLs
```
I'd remove this part as it's anyway not complete, colon and auth part are missing. But should be clear with the following example.

Reply:
Fixed in 7742323ad8183b0785fdfdae5e9530325365a897